docs/release-notes/release-notes-1.6.0-rc1.1.md

```diff
@@ -0,0 +1,925 @@
+# dcrd v1.6.0-rc1.1
+
+This release of dcrd introduces a large number of updates.  Some of the key
+highlights are:
+
+* A new consensus vote agenda which allows the stakeholders to decide whether or
+  not to activate support for a decentralized treasury
+* Aggregate fee transaction selection in block templates (CPFP)
```

Comment:
I'd use the full Child Pays For Parent here.

Reply:
Done.